### DISCUSSION TITLE
How to get Cargo command neo-test is generating?

### DISCUSSION THREAD

**Discussion by nickvikeras** [mrcjkb/rustaceanvim#452] [Q&A] [answered]
I installed Rustacean using the default LazyVim LazyExtra configuration. Is there any easy way to log out the Cargo command neo-test is executing to run unit tests? I just want to be able to copy-paste it into a terminal to re-run the tests outside of vim.

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

Rustaceanvim doesn't provide an API for this. 
I don't know if neotest does. 
You might be able to see it in the neotest logs.
See [`require("neotest.logging"):get_filename()`](https://github.com/nvim-neotest/neotest/blob/32ff2ac21135a372a42b38ae131e531e64833bd3/lua/neotest/logging.lua#L107).
